src/operation/decision_model_optimizer_results.jl

```diff
@@ -0,0 +1,84 @@
+"""
```

Comment:
Since this struct is used within an InMemoryModelStore and InMemorySimulationStore, I think it merits its own file.